locales/en/messages.json

```
@@ -70,5 +70,9 @@
     "content":"",
     "content_youth_risk_assessment":"The $t({{context}}) for <strong>{{name}}</strong> has been successfully confirmed. You can now start the Person Escort Record.",
     "content_person_escort_record":"The handover for <strong>{{name}}</strong> has been recorded. You can view the details recorded in the <a href=\"{{timelineUrl}}\">history of events</a>."
+  },
+  "events_added": {
+    "heading": "Lockout events added successfully",
+    "content": "You have added new lockout events for <b>{{fullName}}</b>. You can view the details in the <a href='#tab-history-of-events' class='govuk-link'>history of events</a>."
```

```suggestion
    "content": "You have added new lockout events for <strong>{{fullName}}</strong>. You can view the details in the <a href='#tab-history-of-events' class='govuk-link'>history of events</a>."
```

I think generally we use a `strong` tag as it's less about the presentation and instead that's controlled in CSS - and it's better for accessibility.